LastHitMarker/Program.cs

```diff
@@ -68,19 +75,34 @@ private static void Game_OnUpdate(EventArgs args)
         private static void Drawing_OnDraw(EventArgs args)
         {
             if (!Game.IsInGame || Game.GameTime > 1800)
+            {
+                if (CreepsDictionary.Count > 0) CreepsDictionary.Clear();
+                if (CreepsTeamDictionary.Count > 0) CreepsTeamDictionary.Clear();
```

Comment:
If a player leave in the middle of a game (example: Bot game) and there is creep in this dictionary, then it will generate error in the next game. Count is a cached variable, thus it won't really affect the performance.